harmony_netcdf_to_zarr/process_utilities.py

```diff
@@ -0,0 +1,30 @@
+from multiprocessing import Process
+from multiprocessing.managers import Namespace
+from time import sleep
+from typing import List
+
+
+def monitor_processes(processes: List[Process], shared_namespace: Namespace, error_notice: str) -> None:
+    """Monitor multiprocess processes for errors.
+
+    Run and monitor multiprocessing processes ensure successful exits.
+    """
+    for process in processes:
+        process.start()
+
+    while any(process.is_alive() for process in processes):
+        sleep(.5)
```

Comment:
No idea what this sleep time should be.  Half a second is pretty often, but this shouldn't cause a lot of overhead